testing/kokoro/system_tests.sh

```diff
@@ -166,7 +167,11 @@ runTests() {
   set +x
   echo "Running 'go test' in '$(pwd)'..."
   set -x
-  2>&1 go test -timeout $TIMEOUT -v "${1:-./...}" | tee sponge_log.log
+  if [[ -f "run_tests" ]]; then
```

Comment:
Looks like the other comment got lost on a previous commit, so I can't seem to continue that conversation. But, please update this to be done purely in the Go test. You can configure environment variables with `os.Getenv`, `os.Setenv`, and `os.Unsetenv`. You can do that directly or using a table driven test (https://github.com/golang/go/wiki/TableDrivenTests).

Reply:
Thanks for this recommendation. I've just implemented and pushed the change :)